Dockerfile

```diff
@@ -98,6 +98,17 @@ RUN chmod +x init.sh
 
 ENTRYPOINT ["/bin/sh", "./init.sh"]
 
+# Frontend builder
+
+FROM inventree_base as frontend
+
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - && apt-get install -y nodejs npm && apt-get autoclean && apt-get autoremove && \
```

Comment:
Installing nodejs and npm is as easy as just doing `apk add nodejs npm` (apt is not available, as the base image now is alpine). See official nodejs [install docs](https://nodejs.org/en/download/package-manager#alpine-linux) for alpine. Same issue below at the dev target.

Reply:
replace them